daemon/logger/jsonfilelog/read_test.go

```diff
@@ -166,7 +166,7 @@ func TestFollowLogsWithRotation(t *testing.T) {
 					// set of logs that loggertest writes.
 					info.Config = map[string]string{
 						"compress": strconv.FormatBool(cmprs),
-						"max-size": "4096b",
+						"max-size": "512kb",
```

Comment:
This max-size is too big for `TestFollowLogsWithRotation` to test across a log-file rotation. Increasing it above 4096b invalidates the test.